loaders missing from core package @0.6.0-rc.1

I just upgraded my app from v0.4.0 to v0.6.0-rc.1 and now I get this error:

```
Failed to compile.

./src/index.js
Module not found: Error: Can't resolve '/Users/guilhermepacheco/Workspace/atitude/asd/node_modules/@redwoodjs/core/dist/loaders/routes-auto-loader' in '/Users/guilhermepacheco/Workspace/atitude/asd/web'
```

The same happens on a fresh app created using `yarn create redwood-app` and upgraded to 0.6.0-rc.1.

I know we still don't have the changelog/upgrade instructions, just wanted to keep track here until we have those.

--- a/packages/core/config/webpack.common.js
+++ b/packages/core/config/webpack.common.js
@@ -164,7 +164,7 @@ module.exports = (webpackEnv) => {
           use: {
             loader: path.resolve(
               __dirname,
-              '../dist/loaders/routes-auto-loader'
+              '../dist/loaders/routes-auto-loader.js'
             ),
           },
         },
@@ -174,7 +174,7 @@ module.exports = (webpackEnv) => {
           test: /.+Cell.(js|tsx)$/,
           include: path.join(redwoodPaths.base, 'web/src/components'),
           use: {
-            loader: path.resolve(__dirname, '../dist/loaders/cell-loader'),
+            loader: path.resolve(__dirname, '../dist/loaders/cell-loader.js'),
           },
         },
       ],

--- a/packages/core/package.json
+++ b/packages/core/package.json
@@ -59,7 +59,7 @@
   },
   "gitHead": "2801c132f40263f9fcfbdac8b1750d2e423eb649",
   "scripts": {
-    "build": "yarn cross-env NODE_ENV=production babel src -d dist --delete-dir-on-start",
+    "build": "yarn cross-env NODE_ENV=production babel src --out-dir dist  --extensions \".ts\" --delete-dir-on-start",
     "prepublishOnly": "yarn build",
     "build:watch": "nodemon --watch src --ext 'js,ts,tsx' --ignore dist --exec 'yarn build'"
   }

